src/pothole_detection/YOLOv4/PyTorch_YOLOv4-train/utils/general.py

```diff
@@ -60,7 +62,8 @@ def check_img_size(img_size, s=32):
     # Verify img_size is a multiple of stride s
     new_size = make_divisible(img_size, int(s))  # ceil gs-multiple
     if new_size != img_size:
-        print('WARNING: --img-size %g must be multiple of max stride %g, updating to %g' % (img_size, s, new_size))
+        print('WARNING: --img-size %g must be multiple of max stride %g, updating to %g' %
```

Comment:
Thank you for wrapping the line! In general, I think f-strings are more readable